components/QuizCardPage/UserPoints/index.test.js

```diff
@@ -1,11 +1,9 @@
 import { render, screen } from "@testing-library/react";
 import { StyledPoints } from ".";
 
-describe("StyledPoints1 Component", () => {
-  test("should render the correct number of points", () => {
-    const points = 42;
-    render(<StyledPoints>{points} Points</StyledPoints>);
-    const points1Element = screen.getByText(`${points} Points`);
-    expect(points1Element).toBeInTheDocument();
-  });
+test("should render the correct number of points", () => {
+  const points = 3;
+  render(<StyledPoints>{points} Points</StyledPoints>);
```

Comment:
This is just a styled component (a `div` with some styling), not a component. This test should be moved to `User/index.test.js` where you actually can check that the passed prop is displayed correctly.